### PR TITLE
WIP: Created course about form

### PR DESCRIPTION
**Pull request by waheedahmed**
ECOM-4885

WIP but comments are welcome.

CC: @bderusha @awais786

### CODE REVIEW

File: course_discovery/apps/publisher/urls.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""
+URLs for the course builder views.
```
Comment (bderusha) [inline course_discovery/apps/publisher/urls.py:2]: publisher views

File: course_discovery/apps/publisher/views.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+"""
+Course Builder views.
```
Comment (bderusha) [inline course_discovery/apps/publisher/views.py:2]: publisher views

File: course_discovery/templates/publisher/course_run_form.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                                        <span class="field-message-content">
+                                            {{ error|escape }}
+                                        </span>
+                                    </div>
+                                {% endfor %}
+                            </div>
+                        {% endfor %}
+                    {% endfor %}
+                </fieldset>
+                <button class="btn-brand btn-base" type="submit">Submit</button>
```
Comment (bderusha) [inline course_discovery/templates/publisher/course_run_form.html:29]: Save

Comment (bderusha) [inline course_discovery/templates/publisher/course_run_form.html:29]: Also translate

File: course_discovery/apps/publisher/models.py [hunk 4]
```diff
@@ -21,16 +21,19 @@ class ChangedByMixin(object):
 class Course(TimeStampedModel, ChangedByMixin):
     """ Publisher Course model. It contains fields related to the course intake form."""
 
-    title = models.CharField(max_length=255, default=None, null=True, blank=True)
-    number = models.CharField(max_length=50, null=True, blank=True)
-    short_description = models.CharField(max_length=255, default=None, null=True, blank=True)
-    full_description = models.TextField(default=None, null=True, blank=True)
+    title = models.CharField(max_length=255, default=None, null=True, blank=True, verbose_name='Course title')
```
Comment (bderusha) [inline course_discovery/apps/publisher/models.py:24]: translate all verbose names

File: course_discovery/apps/publisher/views.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+"""
+from django.views.generic import TemplateView
+from course_discovery.apps.publisher.forms import CourseForm, CourseRunForm
+
+
+class CourseAboutView(TemplateView):
```
Comment (bderusha) [inline course_discovery/apps/publisher/views.py:8]: Can we come up with a better name for this?

File: course_discovery/apps/publisher/forms.py [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    """ Course form."""
+
+    class Meta:
+        model = Course
+        fields = '__all__'
+        exclude = ('organizations',)
+
+    def __init__(self, *args, **kwargs):
+        super(CourseForm, self).__init__(*args, **kwargs)
+        for field_name, field in self.fields.items():
```
Comment (bderusha) [inline course_discovery/apps/publisher/forms.py:16]: Let's put this in a base class and share it

**Comment by bderusha** [comment]
@waheedahmed First pass complete.  General design looks good.  I think that we are going to want more control of the template that just iterating over the fields and spitting them out, but that's something we can do when we get there

**Comment by waheedahmed** [comment]
Closing in favor of https://github.com/edx/course-discovery/pull/174.

**Pull request closed without merging**